tests/graph/test_mermaid.py

```diff
@@ -194,6 +195,86 @@ def test_mermaid_code_all_nodes():
 """)
 
 
+# Tests for direction starts here
+def test_mermaid_code_all_nodes_no_direction():
+    assert graph3.mermaid_code() == snapshot("""\
+---
+title: graph3
+---
+stateDiagram-v2
+  AllNodes --> AllNodes
+  AllNodes --> Foo
+  AllNodes --> Bar
+  Foo --> Bar
+  Bar --> [*]\
+""")
+
+
+def test_mermaid_code_all_nodes_with_direction_lr():
+    assert graph3.mermaid_code(direction='LR') == snapshot("""\
+---
+title: graph3
+---
+stateDiagram-v2
+  direction LR
+  AllNodes --> AllNodes
+  AllNodes --> Foo
+  AllNodes --> Bar
+  Foo --> Bar
+  Bar --> [*]\
+""")
+
+
+def test_mermaid_code_all_nodes_with_direction_tb():
+    assert graph3.mermaid_code(direction='TB') == snapshot("""\
```

Comment:
I don't think we need to test every value, one `direction` value test is enough.